Dockerfile

```diff
@@ -122,6 +122,8 @@ RUN cohortextractor --help
 # Development image that includes test dependencies and mounts the code in
 FROM cohortextractor-base as cohortextractor-dev
 
+RUN --mount=type=cache,target=/var/cache/apt /root/docker-apt-install.sh libmagic1
```

Comment:
How much is libmagic actually getting us here, given that it doesn't understand either feather or dta? It seems like we're basically just using it to confirm that gzipped data is actually gzipped, which we could do with stdlib code if we wanted to, right?

Reply:
fair point, it makes the test a bit awkward, but probably worth it